static/src/javascripts/projects/commercial/modules/brazeBanner.js

```diff
@@ -83,7 +84,67 @@ const canShowPreChecks = ({
 let messageConfig: InAppMessage;
 let appboy: ?AppBoy;
 
+const getForcedBrazeMessage = (): InAppMessage | null => {
+        const params = getUrlVars();
+        const value = params['force-braze-message'];
+        if (value) {
+            try {
+                const dataFromBraze = JSON.parse(value);
+
+                return {
+                    extras: dataFromBraze,
+                };
+            } catch (e) {
+                // Parsing failed. Log a message and fall through.
+                console.log(
+                    'There was an error with force-braze-banner: ',
```

Comment:
Should this be 'an error with force-braze-message'?

Reply:
Yes, definitely! Good spot. Perhaps I could extract the query string arg key to variable and use that here, then it would be impossible to get out of sync.